src/grid/column.component.ts

```diff
@@ -81,7 +81,7 @@ export class IgxColumnComponent implements AfterContentInit {
     public maxWidth: string;
 
     @Input()
-    public minWidth = "48";
+    public minWidth = this.defaultMinWidth;
```

Comment:
Not for this PR: As you already fixed the header to discard the extra space for icons when they are missing, we could now properly calculate the minWidth and not hardcoding it. We could improve this in the future.